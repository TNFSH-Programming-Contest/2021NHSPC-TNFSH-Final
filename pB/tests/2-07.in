8 10 3
3 8 10 6 1 2
4 6 2 5 1 3
5 1 9 4 10 6
4 10 6 1 3 5
7 8 1 6 4 10
8 5 3 9 7 10
8 6 10 7 4 1
4 2 10 3 1 8
7 7 5 8 2 3 1 4
3 2 1 8
5 1 8 2 6 4
6 8 2 3 4 5 7
4 4 2 6 3
6 5 3 2 1 7 4
3 7 5 6
5 7 6 5 1 8
2 6 3
7 8 4 1 5 3 7 6
