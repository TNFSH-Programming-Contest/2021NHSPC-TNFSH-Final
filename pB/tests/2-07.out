0
0
1 1
3 2 4 8
1 3
0
1 5
2 6 7
0
0
